src/rafts/lighthouse/commands/abstract.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable prettier/prettier */
```

Comment:
Avoid disabling eslint rules, especially for entire files, and prettier should never be disabled
```suggestion
```